processor/src/main/java/org/mapstruct/ap/internal/model/MethodReference.java

```diff
@@ -348,4 +350,14 @@ public static MethodReference forMethodCall(String methodName) {
         return new MethodReference( methodName, null, false );
     }
 
+    @Override
+    public String toString() {
```

Comment:
I am not sure if the `toString` is always correct. Is `returnType` always non-null? Looking at the code it is not always like that.

Reply:
good catch!